libcaliptra/src/caliptra_api.c

```diff
@@ -303,8 +334,37 @@ int caliptra_upload_fw(struct caliptra_buffer *fw_buffer)
     if (fw_buffer == NULL)
         return -EINVAL;
 
-    const uint32_t FW_LOAD_CMD_OPCODE = 0x46574C44u;
-    return caliptra_mailbox_execute(FW_LOAD_CMD_OPCODE, fw_buffer, NULL);
+    return caliptra_mailbox_execute(OP_CALIPTRA_FW_LOAD, fw_buffer, NULL);
+}
+
+int caliptra_get_idev_csr_rom(struct caliptra_buffer *buffer)
+{
+    // TODO: Identify if runtime is active and fail
+    // if (caliptra_runtime_active())
+    //     return -EINVAL;
```

Comment:
not for this PR but would it be too complex to maintain a lookup table of commands supported by ROM/RTFW
caliptra_cmd_table = [ {Command_ID, is_supported_by_rom, is_supported_by_rtfw} , ...]
and have a common prefix for each command to check we are in right state to execute that particular command ?